bot/exts/holidays/valentines/lovecalculator.py

```diff
@@ -90,7 +90,7 @@ def normalize(arg: Member) -> Coroutine:
             name="A letter from Dr. Love:",
             value=data["text"]
         )
-        embed.set_footer(text=f"You can unsubscribe from lovefest by using {Client.prefix}lovefest unsub")
+        embed.set_footer(text="You can unsubscribe from lovefest by using !subscribe.")
```

Comment:
`!` is a constant: https://github.com/python-discord/sir-lancebot/blob/42163c1b57047b8f80f6dcdd93ff4602cae26979/bot/constants.py#L39

Reply:
Oh awesome, switched to using that in https://github.com/python-discord/sir-lancebot/pull/1020/commits/3b0217eb43c8cf01dc0d387ff64639d3f1cfe2fe.